rust/datafusion/src/datasource/parquet.rs

```diff
@@ -41,10 +41,25 @@ impl ParquetTable {
     pub fn try_new(path: &str) -> Result<Self> {
         let parquet_exec = ParquetExec::try_new(path, None, 0)?;
         let schema = parquet_exec.schema();
+
+        let metadata = parquet_exec.metadata();
+        let (num_rows, total_byte_size) = metadata.row_groups().iter().fold(
```

Comment:
I think this would be better as two "loops", than it can use `.sum()` as well instead of fold?

Reply:
updated to two iterators